tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -155,6 +155,8 @@ def test_positive_provision_rhev_with_host_group(rhev, provisioning, tear_down):
 
     :expectedresults: The host should be provisioned with host group
 
+    :BZ: 1777992
```

Comment:
This BZ also has a customer case associated with it, so you can set `:customer_scenario:` token as well

Reply:
Done, thanks